examples/step-67/doc/builds-on

```diff
@@ -1 +1 @@
-step-33 step-48 step-59 step-40
+step-33 step-48 step-59
```

Comment:
You'll have to take this change out of the patch -- it's already tracked by #10375 .